ushahidi/platform-client
# 2105 display name

This pull request makes the following changes:
- Always make the save-button enabled on user-create/user-edit
- Displays error-messages correctly when clicking on save
- Adds test for new save-scenario

Testing checklist:
- Go to settings->users->Add a new user
- Click save without adding any information
- [ ] The button should not be disabled
- [ ] You should stay on the page
- [ ] Error-labels where information is missing should be visible
- Add information in the boxes
- Click on save
- [ ] The button should not be disabled
- [ ] You should get redirected to the users-list
- [ ] The user you added should be saved

-Go to settings->users
- Click on a random user
- Empty one or many of the input-fields
- [ ] You should see error labels connected to those fields
- Click on save 
- [ ] The button should not be disabled
- [ ] You should stay on the page
- [ ] Error-labels where information is missing should be visible

- Add information in the boxes
- Click on save
- [ ] The button should not be disabled
- [ ] You should get redirected to the users-list
- [ ] The information you changed should be saved




Fixes ushahidi/platform#2105 .

Ping @ushahidi/platform
